ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -149,12 +148,13 @@ - (void)removeReactSubview:(id<RCTComponent>)subview {
     // underlying mapview action here.
     if ([subview isKindOfClass:[RCTMapboxAnnotation class]]) {
         RCTMapboxAnnotation * annotation = (RCTMapboxAnnotation *) subview;
-        [_reactSubviews removeObjectForKey:annotation.reuseIdentifier];
+        [_map removeAnnotation:annotation];
+        [_reactSubviews removeObject:annotation];
     }
 }
 
 - (NSArray<id<RCTComponent>> *)reactSubviews {
-    return nil;
+    return _reactSubviews;
```

Comment:
Changing this causes the annotation to render in front of the userLocation indicator. I tested switching this back to nil, and the annotation rendered behind the userLocation as expected. I don't know what implications changing this back has, but I think it should be looked in to. Otherwise this PR solves a huge issue for my team.